### PR TITLE
Nuke union merge

### PR DESCRIPTION
**Pull request by anaisbetts**
This reverts our custom `gitattributes` that did union merges on VS project files.

Originally, the idea was, that while this strategy is more often wrong (i.e. not what the user wanted), it also made it so that the user could fix it in Visual Studio (i.e. it didn't put conflict markers in an XML file). 

However, this part isn't often even true, because it will leave off closing tags for elements, which means that VS often can't open it anyways. RIP sanity.

/cc @github/desktop

### CODE REVIEW

**Comment by haacked** [comment]
Well this is a screwed if we do and screwed if we don't scenario. Man, we should pay someone to write that project merge strategy or something.

File: etc/gitattributes [hunk 1]
```diff
@@ -1,11 +1,3 @@
-# Custom for Visual Studio
-*.cs     diff=csharp
```
Comment (haacked) [inline etc/gitattributes:2]: Why is this changing? Seems to have nothing to do with the merge=union.

Comment (anaisbetts) [inline etc/gitattributes:2]: Good catch, I fucked up

**Comment by jspahrsummers** [comment]
Yup, Xcode conflicts are terrible too. IDEs suck.

**Pull request merged**